torch/CMakeLists.txt

```diff
@@ -344,6 +343,10 @@ endif()
 
 if(USE_CUDA)
   if(MSVC)
+    set(NVTOOLEXT_HOME "C:/Program Files/NVIDIA Corporation/NvToolsExt")
```

Comment:
This hardcoded path makes me nervous... :-/ Any way around it. @davidbrownellMS and @peterjc123 any thoughts?

Reply:
I think it should be okay. This path is rather constant. If we want to be safe, then we'll need to do reg query.